src/App.js

```diff
@@ -1,17 +1,42 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import ConferenceList from "./ConferenceList";
+import { getConferenceList } from "./api";
 import styled from "styled-components";
-import conferences from "./conferenceDataMock.json";
 
 const StyledApp = styled.div`
   text-align: center;
   background: lightgrey;
 `;
 
 function App() {
+  const [isLoading, setIsLoading] = useState(false);
+  const [conferences, setConferences] = useState([]);
+  const [errorCaught, setErrorCaught] = useState(false);
+
+  useEffect(() => {
+    async function fetchData() {
+      setIsLoading(true);
+
+      try {
+        const conferences = await getConferenceList();
+        setConferences(conferences);
+      } catch (error) {
+        setErrorCaught(true);
+      }
+
+      setIsLoading(false);
+    }
+    fetchData();
+  }, []);
+
   return (
     <StyledApp>
-      <ConferenceList conferences={conferences} />
+      {isLoading ? (
+        <p>Loading...</p>
+      ) : (
```

Comment:
this is just a readability thing for me, so don't feel the need to do it, but I quite like organising ternarys like this:

```
isLoading
? <p>Loading...</>
: <ConferenceList conferences={conferences} />
```